src/licensedcode/data/rules/afpl-8.0_url_4.RULE

```diff
@@ -0,0 +1,8 @@
+Please note that the program Triangle, 
+written by Jonathan Shewchuk and used by FEMM
+for the purposes of mesh generation is NOT covered
+by this license and is subject to its own licensure terms. 
```

Comment:
You should break this two smaller rules:

```
mesh generation is NOT covered by this license and is subject to its own  terms.
```
may be enough for one rule with a `minimum_coverage: 100`as this is rather unique. I have second thoughts about adding things like "licensure"  as each new added word has an impact with this suggestion there is no "licensre" anymore.

Also `distributed under its own licensing terms.  The licensing terms of Lua are also attached.` would then be  enough for an `other-permissive ` rule